### ISSUE TITLE
[Question] Custom Search Engine to the launcher

### ISSUE THREAD

**Issue by huseyinkogo** [HelloVolla/volla-os-beta-test#145]
I would like to use my self-hosted search engine, on the launcher.

So I have tried sharing the URL below to 'Volla', couldn't suceed.

`https://google.doamin.com/search?q=`

![image](https://github.com/HelloVolla/volla-os-beta-test/assets/76049382/390dfb06-d2b5-4190-9eda-9de9b5bd8317)

**Comment by wurzer** [comment]
Currenlty the search engines are hard coded. You are welcome to contribute to the project. Anyway I will take this ticket as a feature request to add custom search engines. However we wouldn't accept a Google search engine as one of the predefines ones in the Volla launcher, because of the privacy issues. Do you know, that StartPage is unsing the Google search index? It's a kind of proxy to protect the peoples privacy.

**Comment by huseyinkogo** [comment]
@wurzer yeah it would be great for the future to add custom search engine.

I just gave a example as the google.domain.com, its a just SearxNG instance
